internal/git/util_test.go

```diff
@@ -54,16 +55,28 @@ func TestMain(m *testing.M) {
 	}
 	pool = p
 
+	relativePath := "../../tests/e2e-gitserver"
+	absolutePath, err := filepath.Abs(relativePath)
+	if err != nil {
+		log.Fatalf("Failed to get absolute path: %s", err)
+	}
+
+	authorizedKeys := fmt.Sprintf("%s/authorized_keys:/root/.ssh/authorized_keys", absolutePath)
+	httPasswd := fmt.Sprintf("%s/.htpasswd:/auth/.htpasswd", absolutePath)
```

Comment:
Curious - what's actually in those files? Does the user need to provide a password or ssh key?

Reply:
+1